packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -138,38 +154,44 @@ const ModalContent = forwardRef((props, ref) => {
       }
       <div
         {...customProps}
-        tabIndex={platformIsiOS ? '-1' : '0'}
+        tabIndex={platformIsiOS || isCalledFromNotificationDialog ? '-1' : '0'}
         aria-label={ariaLabel}
+        aria-labelledby={ariaLabelledBy}
+        aria-describedby={ariaDescribedBy}
         className={modalClassName}
         role={role}
         ref={ref}
       >
         <div className={modalContainerClassName} ref={setModalFocusElementRef} data-terra-abstract-modal-begin tabIndex="-1">
-          <FormattedMessage id="Terra.AbstractModal.BeginModalDialog">
-            {text => {
-              // In the latest version of react-intl this param is an array, when previous versions it was a string.
-              let useText = text;
-              if (Array.isArray(text)) {
-                useText = text.join('');
-              }
-              return (
-                <VisuallyHiddenText text={useText} />
-              );
-            }}
-          </FormattedMessage>
+          {(!isCalledFromNotificationDialog && isCalledFromNotificationDialog !== undefined) && (
```

Comment:
with default value false, you will able to reduce the above condition to 
```suggestion
          {(!isCalledFromNotificationDialog) && (
```

Reply:
Done [5df6594](https://github.com/cerner/terra-framework/pull/1894/commits/5df65942439721cd6bd96dc4dc96ad8e7ff4acf2)